entity-types/infra-awsdmshost/dashboard.json

```diff
@@ -0,0 +1,443 @@
+            },
+            "thresholds": [
+              {
+                "alertSeverity": "WARNING",
+                "value": 10000
+              }
+            ]
+          }
+        },
+        {
+          "title": "Memory Usage",
+          "layout": {
+            "column": 1,
+            "row": 6,
+            "width": 4,
+            "height": 3
+          },
+          
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT average(aws.dms.MemoryUsage)*1000000 AS 'Memory Usage' timeseries auto facet `aws.dms.ReplicationInstanceIdentifier`"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "units": {
+              "unit": "BYTES"
+            }
+          }
+        },
+        {
+          "title": "CDC Latency Source",
+          "layout": {
+            "column": 5,
+            "row": 6,
+            "width": 4,
+            "height": 3
+          },
+          
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT average(aws.dms.CDCLatencySource) AS 'CDC Latency Source' timeseries auto facet `aws.dms.ReplicationInstanceIdentifier`"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "units": {
+              "unit": "SECONDS"
+            }
+          }
+        },
+        {
+          "title": "CDC Throughput Bandwidth Target (KB per second)",
+          "layout": {
+            "column": 9,
+            "row": 6,
+            "width": 4,
+            "height": 3
+          },
+          
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT average(aws.dms.CDCThroughputBandwidthTarget) AS 'CDC Throughput Bandwidth Target' timeseries auto facet `aws.dms.ReplicationInstanceIdentifier`"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "variables": []
```

Comment:
Could you remove the variables[], our validator currently don't allow them.

Reply:
sure